scylla_shard_aware_port_integration_test.go

```diff
@@ -0,0 +1,24 @@
+// +build integration
+// +build scylla
+
+package gocql
+
+import "testing"
+
+func TestShardAwarePortIntegrationNoReconnections(t *testing.T) {
+	testShardAwarePortNoReconnections(t, func() *ClusterConfig {
+		return createCluster()
+	})
```

Comment:
```suggestion
	testShardAwarePortNoReconnections(t, createCluster)
```

Reply:
?

Reply:
Unfortunately we can't do that (I tried). `createCluster` is a variadic function and we can't cast it to `func() *ClusterConfig`.